helm-chart/values.yaml

```diff
@@ -408,7 +408,7 @@ clickhouse:
   shards: 1
   replicaCount: 1
   extraEnvVarsSecret: cvat-analytics-secret
-  initdbScriptsSecret: cvat-clickhouse-init
+  startdbScriptsSecret: cvat-clickhouse-start
```

Comment:
It seems that a separate job needs to be added for initialization in case of external clichouse use. like this https://github.com/cvat-ai/cvat/blob/develop/helm-chart/templates/cvat_backend/initializer/job.yml